test/test_spsc_queue.ml

```diff
@@ -22,14 +22,14 @@ let test_full () =
   print_string "test_spsc_queue_full: ok\n"
 
 let test_parallel () =
-  let count = 100_000 in
+  let count = 1000 in
```

Comment:
It would be nice to keep the higher number in general case. How about something like this: 

```
let count = 
  let ocaml_4 = Char.code (String.get Sys.ocaml_version 0) < Char.code '5' in
  if ocaml_4 then 1_000 else 100_000 
in 
```

Reply:
Good idea. Done!